packages/eslint-plugin-pf-codemods/src/rules/v6/emptyStateHeaderMoveIntoEmptyState/emptyStateHeader-move-into-emptyState.ts

```diff
@@ -269,10 +236,12 @@ module.exports = {
 
           const titleText =
             titleTextPropValue ||
-            `titleText=${getChildrenAsAttributeValueText(
-              context,
-              header.children
-            )}`;
+            (hasChildren
+              ? `titleText=${getChildrenAsAttributeValueText(
+                  context,
+                  header.children
+                )}`
+              : "");
```

Comment:
And one nit here: can you refactor this in a simpler way than having a ternary nested inside of an or statement?